specification/metadata/metadata_provider.md

```diff
@@ -0,0 +1,38 @@
+# FOCUS Metadata Provider
+The FOCUS metadata provider logical section provides metadata about the provider of the FOCUS data. 
+
+
+## Logical Section: Schema Provider
+
+## Schema Definition Metadata
+
+* ### billing_generator
+    * use: Human readable name of the entity that is generating the billing data 
+    * type: STRING
+* ### provider_prefix
+    * use: Shortened name of the provider supplied to be used as the prefix for tags and may be used to modify the name of custom columns.
+    * type: STRING
+
+
+
+
+## Example Provider Metadata
```

Comment:
What does everyone feel about these API examples. Part of me wants them moved to supporting content and not be in the first cut of this section for 1.0?

Reply:
I am fine moving them as long as we include clear links to the examples.